clients/drcachesim/launcher.cpp

```diff
@@ -323,7 +323,8 @@ _tmain(int argc, const TCHAR *targv[])
             FATAL_ERROR("invalid -outdir %s", op_outdir.get_value().c_str());
         }
     } else {
-        if (op_simulator_type.get_value() == RECORD_FILTER) {
+        if (op_simulator_type.get_value() == RECORD_FILTER ||
+            op_simulator_type.get_value() == INTERNAL_RECORD_VIEW) {
```

Comment:
Do we need the multi-simulator support (colon-separated for memref_t tools) now that we have more than one tool?  Add a XXX comment acknowledging we're aware multi-tool is not currently supported for record tools.